Resources/Prototypes/Entities/Objects/Shields/shields.yml

```diff
@@ -240,32 +243,32 @@
     - type: Blocking
       passiveBlockModifier:
         coefficients:
-          Blunt: 0.95
-          Slash: 0.95
-          Piercing: 0.95
+          Blunt: 0.7
+          Slash: 0.7
+          Piercing: 0.7
       activeBlockModifier:
         coefficients:
-          Blunt: 0.85
-          Slash: 0.85
-          Piercing: 0.85
+          Blunt: 0.7
+          Slash: 0.7
+          Piercing: 0.7
         flatReductions:
-          Blunt: 0.5
-          Slash: 0.5
-          Piercing: 0.5
+          Blunt: 1
+          Slash: 1
+          Piercing: 1
     - type: Construction
       graph: WebObjects
       node: shield
     - type: Destructible
       thresholds:
         - trigger:
             !type:DamageTrigger
-            damage: 40
+            damage: 200
           behaviors:
             - !type:DoActsBehavior
               acts: [ "Destruction" ]
         - trigger:
             !type:DamageTrigger
-            damage: 20
+            damage: 120
           behaviors:
             - !type:DoActsBehavior
               acts: [ "Destruction" ]
```

Comment:
Is it me or does it make no sense for there to be two of these blocks? When it reaches 120 dmg it would be _destroyed_ already. The 200 is unreachable.

I checked the code and it does indeed seem to just break at 120 with this setup.

Reply:
So to what I understand, it seems the second damagetrigger threshold is completely redundant. I think its supposed to be an overflow damage threshold, where the shield will yield NO resources if its outright broken or destroyed by meeting that threshold but I dont want to axe it for fear of breaking the code.

Reply:
No reason not to just try it. If it doesn't crash your game, it's probably fine no? :)